docs/general/quick-start/windows.md

```diff
@@ -0,0 +1,263 @@
+---
+uid: quick-start-windows
+title: Windows
+---
+
+# Windows Quick Start
+
+## Required Tools and Resources
+
+* A Windows 10/11 PC
+* A user account with admin privileges
+* An internet connection
+* A public IP address
+* A text editor (Notepad will work but [Notepad++](https://notepad-plus-plus.org/downloads/) or [VSCode](https://code.visualstudio.com/Download) is recommended)
+* A browser that isn't Internet Explorer
+
+## Before you begin
+
+* Connect your PC with ethernet instead of Wi-Fi
+* Set your PC to never go to sleep automatically ([Guide](https://support.microsoft.com/en-us/windows/how-to-adjust-power-and-sleep-settings-in-windows-26f623b5-4fcc-4194-863d-b824e5ea7679))
+* Show file extensions in Explorer ([Guide](https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/))
+* Setup auto login for your user account if you haven't already ([Guide](https://www.lifewire.com/how-do-i-auto-login-to-windows-2626066))
+* Make your media files accessible on this computer by this user
+* Install latest updates ([Guide](https://support.microsoft.com/en-us/windows/get-the-latest-windows-update-7d20e88c-0568-483a-37bc-c3885390d212))
+* Setup remote access (optional). Popular options include [Teamviewer](https://www.teamviewer.com/), [AnyDesk](https://anydesk.com/), [RustDesk](https://rustdesk.com/), [Google Remote Desktop](https://remotedesktop.google.com/) and [Parsec](https://parsec.app/)
+
+## Notes
+
+* How to open a Terminal from current location in Explorer
+  * Windows 10
+    * Right click on blank space while holding down `Shift`
+    * Click `Open PowerShell window here`
+  * Windows 11
+    * Right click on blank space
+    * Click `Open in Windows Terminal` or `Open in Terminal`
+
+## Jellyfin Install
+
+### Install
+
+1. Download installer exe and sha256sum from [the official repo](https://repo.jellyfin.org/releases/server/windows/stable/). The files to download are `installer/jellyfin_x.y.z_windows-x64.exe` and `installer/jellyfin_x.y.z_windows-x64.exe`.
+
+    ![Windows Download Page](/images/docs/quick-start/windows/jellyfin-0.png)
+
+2. Once the download finishes, open the sha256sum in a text editor
+3. Open a Terminal in your Downloads folder and enter this command: `certutil -hashfile .\jellyfin_x.y.z_windows-x64.exe SHA256`, replace `jellyfin_x.y.z_windows-x64.exe` with the .exe file you just downloaded.
+4. Wait for the command to finish and see if it matches the value in the sha256sum file. If the values don't match, at least one of the files is corrupted and both files should be redownloaded.
+
+    ![Checksum Match](/images/docs/quick-start/windows/jellyfin-1.png)
+
+    Values match, files are good
+
+    ![Checksum Doesn't Match](/images/docs/quick-start/windows/jellyfin-2.png)
+
+    Values don't match, possibly corrupted file(s)
+
+5. Run and follow the installer
```

Comment:
This part might need some more information. Which steps does the installer have, what options do I select (e.g. don't install as a service because that breaks transcoding)